package.json

```diff
@@ -83,23 +74,5 @@
   },
   "keywords": [
     "ern-miniapp"
-  ],
-  "standard": {
-    "env": {
-      "jest": true
-    },
-    "globals": [
-      "before",
-      "describe",
-      "expect",
-      "it"
-    ],
-    "ignore": [
-      "flow-typed"
-    ],
-    "parser": "babel-eslint",
-    "plugins": [
-      "flowtype"
-    ]
-  }
```

Comment:
standard in the upcoming 15 release has some exciting stuff, mainly around better 1st class TS support.  sad to see it go 👋

Reply:
Yeah, true, but fortunately ESLint also has some exciting stuff on the way that should help with better TypeScript support. Mainly the convergence with TSLint: https://medium.com/palantir/tslint-in-2019-1a144c2317a9 :) 👍